DeviceSimulator/DeviceSimulator/AzureIoTHub.cs

```diff
@@ -1,13 +1,12 @@
+using Azure.Messaging.EventHubs.Consumer;
+using Microsoft.Azure.Devices;
 using Microsoft.Azure.Devices.Client;
 using Microsoft.Azure.Devices.Common.Exceptions;
-using Microsoft.ServiceBus.Messaging;
```

Comment:
This was a super old nuget package for event hub. It is so old the package name starts with "WindowsAzure". Our samples should be using nuget packages from this decade. :)

I updated it to the track 2 event hub lib.